task/dev12/src/main/java/com/github/zubarevladimir/Checker/Checker.java

```diff
@@ -0,0 +1,113 @@
+package com.github.zubarevladimir.Checker;
+
+import com.github.zubarevladimir.Validator.CheckerValidator;
+
+public class Checker {
+
```

Comment:
1. Additional class CheckerCoordinate is needed to store coordinates
2. Do not use both String and two int fileds to store coordinate (use only ints, String has to be fomatted on request)
3. Use separate classes for white and black checkers (BlackChecker, WhiteChecker)